### PR TITLE
Add radians to numpy frontend

### LINKED ISSUE #4126: radians

### PR DESCRIPTION
**Pull request by JonaOmara**
closes #4126

### CODE REVIEW

File: ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py [hunk 1]
```diff
@@ -211,3 +211,26 @@ def arctan2(
...
+):
+    if dtype:
+        x = ivy.astype(ivy.array(x), ivy.as_ivy_dtype(dtype))
+    ret = ivy.divide(ivy.multiply(x, ivy.pi, out=out), 180)
+    if ivy.is_array(where):
+        ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
+    return ret
+
+
+radians.unsupported_dtypes = {"torch": ("float16",)}
```
Comment (juliagsy) [inline ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py:236]: I don't think this is unsupported according to the backend, please remove this line

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey there! Thanks for contributing! Just a small change and you're almost good to go! Please request my review when you're done, thank you!

File: ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py (hunk 1 above)
Comment (JonaOmara) [reply ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py]: @juliagsy  thanks for the review, let me remove it

**Review by juliagsy — APPROVED** [review]
Thanks for contributing!

**Pull request merged**